app/controllers/sessions_controller.rb

```diff
@@ -25,7 +25,8 @@ def store_user_info
     # rubocop:disable Rails/SkipsModelValidations
     @user.update_columns(
       provider_token: auth_hash.credentials.token,
-      name: auth_hash.info.nickname
+      name: auth_hash.info.nickname,
+      email: auth_hash.info.email
```

Comment:
So to my mind, it makes sense to set this from the auth hash on user create, since it is also required. Then we are asking folks to update their preferences, which is more in line with the copy anyway.